### PR TITLE
enh: Formatting and colours in messages

### PR DESCRIPTION
**Pull request by csaranbalaji**
* Added code for feature requested in https://github.com/Haider8/tmessage/issues/37
* Formatted code using `black`

### CODE REVIEW

**Comment by Haider8** [comment]
@csaranbalaji How can I test the feature that you have implemented here?

**Comment by csaranbalaji** [comment]
@Haider8 Please test and let me know the changes to be done.

**Comment by Haider8** [comment]
@csaranbalaji Please make a new heading for text formatting in the README and update it for the changes that you have implemented in this pull request.

**Comment by csaranbalaji** [comment]
Yes that is yet to be done thats why mentioned as WIP in PR title :)

**Comment by Haider8** [comment]
@csaranbalaji Sure, make the changes in the README, then I will review and then we can merge this PR

File: requirements.txt [hunk 1]
```diff
@@ -4,4 +4,5 @@ requests==2.22.0
 peewee==3.11.2
 PyJWT==1.7.1
 python-dotenv==0.10.3
-pylint==2.4.2
\ No newline at end of file
+pylint==2.4.2
+simple_chalk
```
Comment (Haider8) [inline requirements.txt]: Add this dependency in `setup.py` also and make the changes as suggested below

```suggestion
simple-chalk==0.1.0
```

Comment (csaranbalaji) [reply requirements.txt]: Done

**Comment by Haider8** [comment]
@csaranbalaji Fix the Travis check

File: docs/features/style_and_colours.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#Message text style and colours
```
Comment (Haider8) [inline docs/features/style_and_colours.md]:
```suggestion
# Message text style and colours
```

File: tmessage/utils.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    def handle_starttag(self, tag, attrs):
+        self.add_tag_from_method_call_list(tag)
+
+    def handle_endtag(self, tag):
+        self.remove_tag_from_method_call_list(tag)
+
+    def handle_data(self, data):
+        if self.method_call_list:
+            method_to_call = "chalk." + ".".join(self.method_call_list)
+            self.message_to_show += eval(method_to_call + '("' + data + '")')  # pylint: disable=eval-used
```
[the remark this answers is not in the document]
Comment (csaranbalaji) [reply tmessage/utils.py]: It became long due to the comment, Should I break the line?

**Comment by Haider8** [comment]
@csaranbalaji PR looks good to go. What can we do with the default green color? It is quite distracting and unreadable when the text is colored.

**Comment by csaranbalaji** [comment]
> @csaranbalaji PR looks good to go. What can we do with the default green color? It is quite distracting and unreadable when the text is colored.

I have removed the green background for the messages received. Isn't that working?

**Comment by Haider8** [comment]
Thank you for this PR. Be sure to check out some other issues and open new ones also :)

**Pull request merged**